src/lxc/af_unix.c

```diff
@@ -56,7 +56,7 @@ int lxc_abstract_unix_open(const char *path, int type, int flags)
 	addr.sun_family = AF_UNIX;
 
 	len = strlen(&path[1]) + 1;
-	if (len >= sizeof(addr.sun_path) - 1) {
```

Comment:
They're not actually null-terminated, only `\0` initiated and use all bytes up to the given length of the address structure. From `unix(7)`: `The socket's address in this namespace is given by the additional bytes in sun_path that are covered by the specified length of the address structure. (Null bytes in the name have no special significance.)`.
If I'm reading this right the current code enforces space for a `\0` while the new code doesn't. (Having a `\0` makes sense if they're checked with `strlen` and seems like a useful convention in general.)